cogs/gaminator.py

```diff
@@ -33,6 +50,32 @@ def option_to_string(self, option):
 
         return f'`@{role_name}`{padding}{qualified_name}'
 
+
+# Hmm... (figure out a means to DRY)
+class DestinyRoleMenu(MenuWithOptions):
+    # Override
+    def update_feedback_ui(self):
+        ctx = self.get_ctx()
+        adding = []
+        removing = []
+        for option in self.get_selected_options():
+            if option[0] not in [role.name for role in ctx.message.author.roles]:
+                adding.append(f'`@{option[0]}`')
+            else:
+                removing.append(f'`@{option[0]}`')
+
+        self.set_feedback_ui_field_at(0, 'Adding', '\n'.join(adding), 'None')
+        self.set_feedback_ui_field_at(1, 'Removing', '\n'.join(removing), 'None')
```

Comment:
Yeah this pattern was more of a stopgap than anything. Since a lot of our widgets are used for role updates maybe we should create a RoleUpdateMenu subclass that handles this statekeeping internally?